src/client/driver/command-executors/execute-action-command.js

```diff
@@ -103,8 +186,24 @@ function createAutomation (elements, command) {
         case COMMAND_TYPE.typeText:
             return new TypeAutomation(elements[0], command.text, command.options);
 
-        case COMMAND_TYPE.hover :
-            return new HoverAutomation(elements[0], command.options);
+        case COMMAND_TYPE.selectText:
+            ensureElementEditable(elements[0]);
```

Comment:
I suppose that `ensureCommandElements` is a better place for it. This check maybe chained to the Promise returned by `ensureElement` [here](https://github.com/DevExpress/testcafe/pull/484/files#diff-527bf138d6eb5bb49021ec4e988a2ccfR130), or `ensureElementEditable` can internally call `ensureElement` and return promise.